boards/nordic/nrf52840dk_opensk/src/main.rs

```diff
@@ -106,7 +108,7 @@ pub mod io;
 // Whether to use UART debugging or Segger RTT (USB) debugging.
 // - Set to false to use UART.
 // - Set to true to use Segger RTT over USB.
-const USB_DEBUGGING: bool = true;
+const USB_DEBUGGING: bool = false;
```

Comment:
With Segger USB debugging, I could not see the debug messages at the same time while using the nordic board as a USB device. This is because, after hitting the "reset" button, the USB connection terminates and RTT won't function anymore because USB is already used by CTAP.